### PR TITLE
[Rule Tuning] Tune Unusual ADS File Creation for Teams weblogs

### LINKED ISSUE #2928: [Rule Tuning] Unusual File Creation - Alternate Data Stream
## Link to rule
[defense_evasion_unusual_ads_file_creation.toml](https://github.com/elastic/detection-rules/blob/main/rules/windows/defense_evasion_unusual_ads_file_creation.toml)

## Description
Teams creates an archive of web logs on a regular basis, and this causes a false positive each time. 

## Example Data
```
{
  "_index": ".ds-logs-endpoint.events.file-default-2023.07.08-000002",
  "_id": "cyN3VokBehJvViqEwjMq",
  "_version": 1,
  "_score": 0,
  "_source": {
    "agent": {
      "id": "01234567-89ab-cdef-fedc-ba9876543210",
      "type": "endpoint",
      "version": "8.8.2"
    },
    "process": {
      "Ext": {
        "ancestry": [
          "NjE4YmQ3ZjYtOWZhYi00ZmJiLTg3ZjgtNTE0NTc4ZGQ2ZjIyLTM2MDAtMTY4ODY1MzE5OS40MjIzMzk0MDA=",
          "NjE4YmQ3ZjYtOWZhYi00ZmJiLTg3ZjgtNTE0NTc4ZGQ2ZjIyLTEwNTU2LTE2ODg2NTMxNzIuMjUxNjM0MDA=",
          "NjE4YmQ3ZjYtOWZhYi00ZmJiLTg3ZjgtNTE0NTc4ZGQ2ZjIyLTEwNTI4LTE2ODg2NTMxNzEuODM5ODgwOTAw",
          "NjE4YmQ3ZjYtOWZhYi00ZmJiLTg3ZjgtNTE0NTc4ZGQ2ZjIyLTgyOC0xNjg4NjUzMTI2Ljk3NjcyNDcwMA=="
        ],
        "code_signature": [
          {
            "trusted": true,
            "subject_name": "Microsoft Corporation",
            "exists": true,
            "status": "trusted"
          },
          {
            "trusted": true,
            "subject_name": "Microsoft Corporation",
            "exists": true,
            "status": "trusted"
          }
        ]
      },
      "parent": {
        "pid": 3600
      },
      "code_signature": {
        "trusted": true,
        "subject_name": "Microsoft Corporation",
        "exists": true,
        "status": "trusted"
      },
      "name": "Teams.exe",
      "pid": 16116,
      "thread": {
        "id": 18064
      },
      "entity_id": "NjE4YmQ3ZjYtOWZhYi00ZmJiLTg3ZjgtNTE0NTc4ZGQ2ZjIyLTE2MTE2LTE2ODg2NTMyMTUuNjkzMzc0NDAw",
      "executable": "C:\\Users\\jdoe\\AppData\\Local\\Microsoft\\Teams\\current\\Teams.exe"
    },
    "message": "Endpoint file event",
    "@timestamp": "2023-07-14T22:13:30.587421Z",
    "file": {
      "Ext": {
        "header_data": [],
        "entropy": 1.208692660492769,
        "header_bytes": "70000000444753545245414d0a050000",
        "windows": {
          "zone_identifier": -1
        },
        "monotonic_id": 528483
      },
      "path": "C:\\users\\jdoe\\appdata\\roaming\\microsoft\\teams\\old_weblogs_20230714161330.txt:6E53BFF5-0001-412b-8407-E3AEDE763511:$DATA",
      "extension": "txt",
      "size": 2830,
      "name": "old_weblogs_20230714161330.txt:6E53BFF5-0001-412b-8407-E3AEDE763511:$DATA"
    },
    "ecs": {
      "version": "1.11.0"
    },
    "related": {
      "hosts": [
        "Computer1"
      ],
      "ip": [
        "192.168.1.10",
        "127.0.0.1"
      ],
      "user": [
        "jdoe"
      ],
      "mac": [
        "01:23:45:67:89:AB",
        "01:23:45:67:89:AC"
      ]
    },
    "data_stream": {
      "namespace": "default",
      "type": "logs",
      "dataset": "endpoint.events.file"
    },
    "elastic": {
      "agent": {
        "id": "01234567-89ab-cdef-fedc-ba9876543210"
      }
    },
    "host": {
      "hostname": "Computer1",
      "os": {
        "Ext": {
          "variant": "Windows 10 Enterprise"
        },
        "kernel": "22H2 (10.0.19045.3086)",
        "name": "Windows",
        "family": "windows",
        "type": "windows",
        "version": "22H2 (10.0.19045.3086)",
        "platform": "windows",
        "full": "Windows 10 Enterprise 22H2 (10.0.19045.3086)"
      },
      "ip": [
        "192.168.1.10",
        "127.0.0.1"
      ],
      "name": "Computer1",
      "id": "01234567-89ab-cdef-fedc-ba9876543210",
      "mac": [
        "01:23:45:67:89:AB",
        "01:23:45:67:89:AC"
      ],
      "architecture": "x86_64"
    },
    "event": {
      "agent_id_status": "verified",
      "sequence": 2359278,
      "ingested": "2023-07-14T22:13:56Z",
      "created": "2023-07-14T22:13:30.587421Z",
      "kind": "event",
      "module": "endpoint",
      "action": "creation",
      "id": "N8PBIo9AgHO7GBpt++++gfvU",
      "category": [
        "file"
      ],
      "type": [
        "creation"
      ],
      "dataset": "endpoint.events.file",
      "outcome": "unknown"
    },
    "user": {
      "domain": "CONTOSO",
      "name": "jdoe",
      "id": "S-1-5-21-012345678-901234567-890123456-78901"
    }
  }
}
```

### PR DESCRIPTION
**Pull request by MakoWish**
## Issues
- Closes #2928 

## Summary

This PR is to whitelist a false-positive for Microsoft Teams archiving web logs. 

## Contributor checklist

- [X] Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- [X] Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/.defense_evasion_unusual_ads_file_creation.toml.swp
Comment (w0rk3r) [inline rules/windows/.defense_evasion_unusual_ads_file_creation.toml.swp:1]: We need to delete this one

**Review by w0rk3r — APPROVED** [review]
LGTM after we remove the .swp file

File: rules/windows/.defense_evasion_unusual_ads_file_creation.toml.swp
Comment (Aegrah) [reply rules/windows/.defense_evasion_unusual_ads_file_creation.toml.swp:1]: ++

**Review by Aegrah — APPROVED** [review]
After deleting the .swp file, LGTM

**Pull request merged**